.env.example

```diff
@@ -1,6 +1,6 @@
 APP_NAME=Laravel
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:qLNLNxel0tiZebepW7pDgFkZLGOq+5ZFi/2uQPr1WSY=
```

Comment:
Please remove this.

Reply:
This uses php artisan key:generate which is required if we are using sanctum:login from csrf-cookie generation. For more info here: https://laravel.com/docs/7.x/installation